lib/Conversion/TosaToTTIR/TosaToTTIRPatterns.cpp

```diff
@@ -81,6 +81,46 @@ class TosaToTTIRMultiplyOpConversionPattern
   }
 };
 
+class TosaToTTIRMatMulOpConversionPattern
```

Comment:
I think we are used to `Matmul` being one-word-ish. Use `Matmul` instead of `MatMul` everywhere.

Reply:
changed